### PR TITLE
Add unit test of aria-expanded to dropdown suite

### PR DESCRIPTION
**Pull request by kkirsche**
Add test for `aria-expanded` to dropdown unit tests

### CODE REVIEW

**Comment by twbs-savage** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: b441658d791683597d4f110beb828e277e7e539a
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/51869037

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by kkirsche** [comment]
Rebased to use `propEqual` instead of `equal` to ensure it is a strict type and value check.

**Comment by twbs-savage** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 0a0d3033a8eea0cf4dcb757b5669b0cc88d4823b
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/51871033

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by cvrebert** [comment]
Perhaps I'm mistaken, but it seems like [`strictEqual`](http://api.qunitjs.com/strictEqual/) would make the most sense.

**Comment by hnrch02** [comment]
`strictEqual` is also what we use elsewhere in the unit tests, FWIW.

File: js/tests/unit/dropdown.js [hunk 1]
```diff
@@ -46,6 +46,23 @@ $(function () {
...
+        + '<li><a href="#">Secondary link</a></li>'
+        + '<li><a href="#">Something else here</a></li>'
+        + '<li class="divider"/>'
+        + '<li><a href="#">Another link</a></li>'
+        + '</ul>'
+        + '</li>'
+        + '</ul>'
+    var $dropdown = $(dropdownHTML).find('[data-toggle="dropdown"]').bootstrapDropdown().click()
+
+    assert.propEqual($dropdown.attr('aria-expanded'), 'true', 'aria-expanded is set to string "true" on click')
```
Comment (hnrch02) [inline js/tests/unit/dropdown.js]: For the sake of consistency, I'd suggest you use the global version of the assert here until we change all of the tests to use the newer style.

Comment (cvrebert) [inline js/tests/unit/dropdown.js]: Meh, I'm already rectifying that: #15893

**Comment by kkirsche** [comment]
Rebased to StrictEqual — sorry used to karma and apparently didn't read down far enough in the QUnit api docs.

**Comment by twbs-savage** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 843bc35bc5a38cc0d5bfd81fcab2b5237680838d
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/51980322

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by kkirsche** [comment]
This broke when rebasing with the updates to QUnit were made. Need to remake this.

**Pull request closed without merging**